knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml

```diff
@@ -783,16 +797,23 @@ spec:
 # limitations under the License.
 
 apiVersion: v1
+data:
+  version: 1.0.0
+  # eventing-kafka.kafka.brokers: Replace this with the URLs for your kafka cluster,
+  #   which is in the format of my-cluster-kafka-bootstrap.my-kafka-namespace:9092.
+  # eventing-kafka.kafka.authSecretName: name-of-your-secret-for-kafka-auth
+  # eventing-kafka.kafka.authSecretNamespace: namespace-of-your-secret-for-kafka-auth
+  eventing-kafka: |
+    kafka:
+      brokers: REPLACE_WITH_CLUSTER_URL
+      topic:
+        defaultNumPartitions: 4
+        defaultReplicationFactor: 1 # Cannot exceed the number of Kafka Brokers!
+        defaultRetentionMillis: 604800000  # 1 week
 kind: ConfigMap
 metadata:
   name: config-kafka
   namespace: knative-eventing
-data:
-  # Broker URL. Replace this with the URLs for your kafka cluster,
-  # which is in the format of my-cluster-kafka-bootstrap.my-kafka-namespace:9092.
-  bootstrapServers: REPLACE_WITH_CLUSTER_URL
-  #authSecretName: name-of-your-secret-for-kafka-auth
-  #authSecretNamespace: namespace-of-your-secret-for-kafka-auth
```

Comment:
we should keep this instead of the `kafka.brokers`

Reply:
already captured as https://issues.redhat.com/browse/SRVKE-812

Will take a look next week ...